edgepi-pwm.js

```diff
@@ -0,0 +1,110 @@
+module.exports = function (RED) {
+  const rpc = require("@edgepi-cloud/edgepi-rpc");
+
+  function PWMNode(config) {
+    RED.nodes.createNode(this, config);
+    const node = this;
+    const pwmPin = rpc.PWMPins[config.pwmPin];
+
+    setInitialConfigs(config, node)
+      .then((pwm) => {
+        node.on("input", async function (msg, send, done) {
+          node.status({ fill: "green", shape: "dot", text: "input received" });
+          try {
+            await updateConfigsOnInput(msg, pwm, config);
+            msg = {
+              payload: await pwm.getDutyCycle(pwmPin),
+              pwmPin: pwmPin,
+              frequency: await pwm.getFrequency(pwmPin),
+              polarity: await pwm.getPolarity(pwmPin),
+              enabled: await pwm.getEnabled(pwmPin),
+            };
```

Comment:
this `msg` is the one to send right? So there's two `msg` variables but one is the received message and the other is the message to send? Is it possible to make the names different so it's more clear what each variable is for and how we intend to use it

Reply:
Hmm. I see what you mean, but I think it might just be a design decision. Other code I've seen tends to use a single msg variable, which is more in line with how node-red has the input and output both called msg.